Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -663,7 +664,17 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Run
                     {
                         foreach (AppAgentMapping appAgentConfig in runnerConfig.AppAgentMappings)
                         {
-                            GingerCore.Platforms.ApplicationAgent appAgent = null;
+                            ApplicationPlatform app = (ApplicationPlatform)FindItemByIDAndName<ApplicationPlatform>(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary cast to 'ApplicationPlatform'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4969846)